### PR TITLE
List of products feature added

### PR DESCRIPTION
**Pull request by SankarVinoth**
- [x ] List of Products Feature added
- [x] Test Case Included

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>in.yourname</groupId>
-	<artifactId>app</artifactId>
+	<groupId>in.SankarVinoth</groupId>
```
Comment (nareshkumar-h) [inline pom.xml]: Change it to lowercase

File: src/main/webapp/ListProducts.jsp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+					<th scope="col">S.no</th>
+					<th scope="col">Product Name</th>
+					<%Map<Integer,String>Products=ProductService.getProducts();
+					int i=0;
+					for(Integer serialNo:Products.keySet()){
+						String productName=Products.get(serialNo);
+						i++;
+						%>
+			      <tr>
+			     <td><%=serialNo%></td>
```
Comment (nareshkumar-h) [inline src/main/webapp/ListProducts.jsp]: Use "i" value instead of serialNo

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -1,20 +1,16 @@
 <%@page import="java.util.Date"%>
```
Comment (nareshkumar-h) [inline src/main/webapp/index.jsp]: Remove unused imports

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**